packages/gasket-plugin-nextjs/README.md

```diff
@@ -184,6 +184,43 @@ module.exports = {
 }
 ```
 
+### nextPreHandling
```

Comment:
What sort of prehandling does this lifecycle solve which could not already be solved by the middleware lifecycle? I don't see any need for it in this PR code, and the given example is not convincing. Is the purpose just to provide access to the `next` instance, and if so, what do you plan to be able to do with it?

Reply:
Well originally I wanted to use it to label transactions I knew were about to be handled by next.js before switching to this other approach. I can remove if you'd prefer to not introduce this.

Reply:
Ok, yea it is an oddball here. Unless we have a strong use-case for it I would prefer not to introduce another lifecycle.

Reply:
Removed.